### DISCUSSION TITLE
CSS not loading

### DISCUSSION THREAD

**Discussion by mbensoussan** [OpenMage/magento-lts#2671] [Questions & Answers]
Hi, after and update of my openmage version (19 to 20.0.16), the css is not loading on the frontend. (but all image are load)
the backend work perfectly. but the frontend call the all.min.css and not the real one.
i have try to : 
- flush all db in redis-cli
- flush media cache
- flush var/cache
- flush database session.
-reconfigure permission : 
find /www/ -type f -exec chmod 644 {} \;
find /www/ -type d -exec chmod 755 {} \;
find /www/var/ -type f -exec chmod 644 {} \;
find /www/media/ -type f -exec chmod 644 {} \;
find /www/var/ -type d -exec chmod 755 {} \;
find /www/media/ -type d -exec chmod 755 {} \;

I have no idea of what happen.

thanks you for you help

**Comment by kiatng** [comment]
I did a google search on `all.min.css`, it seems to be related to font-awesome, which is not part of Magento 1 nor OpenMage core. You can try to download the css into the skin folder.  This [link](https://www.srapsware.com/tech/news/2021/06/21/fontawesome-6/) may help.

**Comment by addison74** [comment]
What errors appear in the browser console?
